akka-stream/src/main/scala/akka/stream/impl/Stages.scala

```diff
@@ -211,6 +212,12 @@ private[stream] object Stages {
     override def withAttributes(attributes: Attributes) = copy(attributes = attributes)
   }
 
+  object Split {
```

Comment:
No need for `final`.
We sometimes use it to mark "intent" more than any kind of correctness.